src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/ValuesColumn.js

```diff
@@ -30,7 +30,14 @@ export const ValuesColumn = ({ action, actionIndex, onChange, option }) => {
 
   const filteredAndMappedNotes = getNotesOptions(formatMessage, itemNotes)
     .filter(obj => obj.value !== option)
-    .map(({ label, value }) => ({ label, value }));
+    .map(({ label, value }) => ({ label, value }))
+    .sort((a, b) => {
+      if (a.label === 'Select option') return -1;
```

Comment:
not sure, that it's good idea to sort by placeholder, because we have translations and it will not work for other langs except english. You just can take first element of array - because we know that placeholder is always firs, or use label.disabled flag, because only placeholders are disabled now

Like this:

```
const sortWithoutPlaceholder = (array) => {
    const first = array.slice(0, 1);
    const rest = array.slice(1).sort((a, b) => a.label.localeCompare(b.label));

    return [...first, ...rest]
}

const sortedNotes = sortWithoutPlaceholder(filteredAndMappedNotes)
```